mkosi.md

```diff
@@ -1515,13 +1529,14 @@ Hostname=image
 
 # REQUIREMENTS
 
-mkosi is packaged for various distributions: Debian, Ubuntu, Arch Linux, Fedora, OpenMandriva.
-It is usually easiest to use the distribution package.
+mkosi is packaged for various distributions: Debian, Ubuntu, Arch
+Linux, Fedora Linux, OpenMandriva. It is usually easiest to use the
+distribution package.
```

Comment:
In Debian mkosi is still at version 5, so one's mileage may vary.

Reply:
It's a package, and it is easy to install. This text doesn't imply anything else ;]